crossbeam-skiplist/src/map.rs

```diff
@@ -248,6 +249,16 @@ impl<'a, K, V> Entry<'a, K, V> {
     }
 }
 
+impl<'a, K, V> Drop for Entry<'a, K, V>
+{
+    fn drop(&mut self) {
+        let guard = &epoch::pin();
+        unsafe {
+            ManuallyDrop::take(&mut self.inner).release(guard);
```

Comment:
Oh, I didn't realize `ManuallyDrop::take` was unstable! :( Fortunately, we can replace it with:

```suggestion
            ManuallyDrop::into_inner(ptr::read(&mut self.inner)).release(guard);
```

Reply:
Thanks! Fixed this in https://github.com/crossbeam-rs/crossbeam/pull/337/commits/7f4f564bc67f275ec43209274d48bb18de394720.